### PR TITLE
Cleanup in IL code

### PR DESCRIPTION
**Pull request by yuvalabou**

### CODE REVIEW

**Comment by yuvalabou** [comment]
Maybe later

**Comment by madsnedergaard** [comment]
Hey @yuvalabou, thanks for your contribution here - would you mind explaining why you closed the PR? :)

**Comment by yuvalabou** [comment]
I have noticed that I can do a more complete work here, and I am now testing new data for production and consumption. I will probably open this pr later when done testing.
However I may have some problems testing this on local docker.
Anyway, will try my best to work on it and learn, as I am no developer whatsoever 🙂

**Comment by yuvalabou** [comment]
Hope this will work

File: parsers/IL.py [hunk 1]
```diff
@@ -7,37 +7,86 @@
...
+        },
+        "source": IEC_URL,
+        "price": PRICE,
     }
 
-    return datapoint
+    return data
+
 
+# def fetch_consumption(zone_key='IL', session=None, target_datetime=None, logger=None):
```
Comment (jarek) [inline parsers/IL.py:72]: delete commented out code here

**Review by FelixDQ — APPROVED** [review]
LGTM!

**Comment by FelixDQ** [comment]
Thanks for the PR @yuvalabou! 
Can you update the branch so that we can merge it?

**Comment by yuvalabou** [comment]
I am not really sure what do you mean, branch is up to date, doesn't it?

**Comment by yuvalabou** [comment]
Oh, that repo was messed up and everything that I tried made it harder to fix so I deleted it 🤦

**Comment by FelixDQ** [comment]
Oh okay, would you mind opening a new PR then? I don't think that we'll be able to merge this

**Pull request closed without merging**